data/changelog.txt

```diff
@@ -173,6 +216,29 @@ Don't allow to scan books that are owned by other characters
 Fix NPC putting items in open air when fetching items during an activity when 3D FOV is on
 Trees and other FLAMMABLE_ASH terrain leaves behind ash when burned down
 Do not report monsters breaking free of unknown grabbers if the player cannot see them
+Hub 01 won't buy back some of their newer stuff
+Fixes https://github.com/CleverRaven/Cataclysm-DDA/issues/66599
+Makes dry duffel bags watertight
+Cleaning BP fouling removes the hidden rust counter
+Fixing XL survivor boots to be the same as other survivor boots
+Reduce airdrop frequency
+Allows certain docks to be placed on non-flowing shallow and deep water.
+Fix the epilogue for the New England Church Community
+Add MAN_MADE flag to dinosaur exhibit
+fix reported bugs in 66637, related to the new smartphone case variants and a changed id, and takes the time to ice the typos recorded in #66620
+Fixed the issue of not properly handling pre existing items in the backpack after installing it on the pet.
+flat armor penetration is spread across all armor layers instead of applying its full value to each
+armor bionics conflict with new mutations and protect against stab damage
+30 gal drum crafts in the correct time
+you can't drink soapy water anymore
+Fix tow cables being unable to connect different vehicles
+Tile rendering fixes
+Add Digitigrade Feet to Extra Mut Scen, fix selecting burrowing
+Fixes typo in the description of the CAR flag
+change dark wyrm's blood type to foul acidic blood
+gold rings don't conflict with metal gauntlets
+Updates the mutation so that players can't see over objects while running on all-fours
+Smashing now incorporates any MELEE_DAMAGE and STRENGTH enchantments
```

Comment:
```suggestion
Allows certain docks to be placed on non-flowing shallow and deep water.
Fix the epilogue for the New England Church Community
Flat armor penetration is spread across all armor layers instead of applying its full value to each
Fix tow cables being unable to connect different vehicles
Smashing now incorporates any MELEE_DAMAGE and STRENGTH enchantments
```